modules/tinymce/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Fixed the dev ZIP missing the required `bin` scripts to build from the source #TINY-8542
 - Fixed a regression whereby text patterns couldn't be updated at runtime #TINY-8540
+- Dialogs will not exceed window height on smaller screens #TINY-8146
```

Comment:
This is in the 6.0.1 section, it should be in the `Unreleased` section above 6.0.1 since it's meant to be for 6.1.0 based on the Jira (the unreleased section currently has not entries).

Reply:
I've gone ahead and moved this so we can get it merged today.